### PR TITLE
Add recently approved events to default newsletter

### PR DESCRIPTION
**Pull request by filipsuk**
So far I had to build newsletter intro text html manually (usually contains new events).

Now fresh newsletter from newsletters:create command contains initial html intro with recently approved events ordered by size and start date. You are still able to do adjustments before send, however the process will be much faster.

### CODE REVIEW

File: app/Modules/Core/Model/EventModel.php [hunk 1]
```diff
@@ -154,4 +154,29 @@ public function findPreviousEvent(int $eventSeriesId): ?IRow
...
+
+    /**
+     * @return Event[]
+     */
+    public function getApprovedEventsByDate(
+        DateTime $from,
+        ?DateTime $to = NULL,
+        bool $orderByRate = false
+    ): array
+    {
```
Comment (jakubzaba) [inline app/Modules/Core/Model/EventModel.php]: Put the bracket on the end of the line above pls.

File: app/Modules/Newsletter/Model/NewsletterService.php [hunk 2]
```diff
@@ -130,7 +140,7 @@ public function createDefaultNewsletter(): IRow
         $newsletter->setSubject($parameters['defaultSubject'] ?? '');
         $newsletter->setFrom($parameters['defaultAuthor']['email'] ?? '');
         $newsletter->setAuthor($parameters['defaultAuthor']['name'] ?? '');
-        $newsletter->setIntroText('');
+        $newsletter->setIntroText($this->renderRecentlyApprovedEvents($this->getApprovedEventsSinceLastNewsletter()));
```
Comment (jakubzaba) [inline app/Modules/Newsletter/Model/NewsletterService.php]: This line is hard to read. Can you format it this way
```
$newsletter->setIntroText(
    $this->renderRecentlyApprovedEvents(
        $this->getApprovedEventsSinceLastNewsletter()
    )
);
```

File: app/Modules/Core/Model/EventModel.php [hunk 3]
```diff
@@ -154,4 +154,29 @@ public function findPreviousEvent(int $eventSeriesId): ?IRow
             ->order('end DESC')
             ->fetch() ?: null;
     }
+
+    /**
+     * @return Event[]
+     */
+    public function getApprovedEventsByDate(
```
Comment (jakubzaba) [inline app/Modules/Core/Model/EventModel.php]: Do you plan to use this method in the near future? How about make it simple as you need. No `$to` or `$orderByRate`?
The usage then would be way simpler.

**Review by jakubzaba — CHANGES_REQUESTED** [review]
Cool, great you started steps to the automation :) Can you take a look to a few format changes?

**Comment by filipsuk** [comment]
@jakubzaba Good points. God I miss prettier.

**Review by jakubzaba — APPROVED** [review]

**Pull request merged**